app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,12 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      Alerting.AlertReset();
+      if(exportData !== undefined && exportData.Keystore !== undefined && exportData.Keystore.length > 0) {
+    	Alerting.AlertAddMsg('top', 'warning', 'Warning: The server\'s TLS private key will be exported and should be protected!');
```

Comment:
let me address this after @napiergit confirms the rest of my reviews here so that i can have one commit instead to have this all resolved!